tests/small/LandIce/SHMIP/input_bueler_unsteady.yaml

```diff
@@ -52,7 +52,8 @@ ANONYMOUS:
         Water Thickness Exponent: 1.0
         Potential Gradient Norm Exponent: 2.0
         Transmissivity: 1.0e-06
-        Regularize With Continuation: false
+        Regularization:
```

Comment:
Can you add a regression sublist?

Reply:
Ah, again, not tested. Since this is the only unsteady input file, I should probably make sure it works (i.e., the solution after a long time is similar to the steady state one), and then add a test.